.github/workflows/docker-image.yml

```diff
@@ -55,7 +62,12 @@ jobs:
         docker info
         docker-compose version
     - name: Build Image
-      run: docker-compose build --build-arg X_LEGACY_GD_LIB=$X_LEGACY_GD_LIB
+      uses: docker/bake-action@v2
+      with:
+        files: docker-compose.yml
+        load: true
+        set: |
+          *.platform=linux/amd64
```

Comment:
That is why tests runs only for amd64, that is why it is quick for pull requests and push events.